### ISSUE TITLE
Frequencies Online

### ISSUE THREAD

**Issue by prosch35k** [minsulander/vatiris#72]
A feature like VATScout where you can see the current online frequencies on and above the aerodome, would be really nice. Would help with handoffs.

**Comment by minsulander** [comment]
Good idea. Should be able to more or less copy-paste the functionality from VatScout.

**Comment by maxlk96** [comment]
@minsulander maybe a point having the same behaviour as in ES, being able to filter, only showing primed positions (using cidsi API maybe).

Unless VatScout already does that?

**Comment by minsulander** [comment]
Yeah I just implemented that the other day in VatScout... it looks at when the frequency is set to 199.998 which appears to me like it's the same thing as being unprimed.
